tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.out

```diff
@@ -0,0 +1,38 @@
+Subject: lucashi fact of the day
+Content-Type: multipart/related; boundary="message"
+
+--message
+Content-Type: multipart/alternative; boundary="body"
+
+--body
+Content-Type: text/html; charset="UTF-8"
+
+<h1>Hello!</h1>
+<p>it is i your dude<br>
+thank you for subscribing to<br>
+lucashi's lit facts</p>
+
+<p>Did <em>you</em> know that
+this lucashi fact is a test fact
+</p>
+<img src="cid:lucas">
+
+--body--
+--message
+Content-Type: image/jpeg; name="lucashi.jpg"
+Content-Disposition: inline; filename="lucashi.jpg"
+Content-Transfer-Encoding: base64
+Content-ID: <lucas>
+X-Attachment-Id: lucas
+
+/9j/4AAQSkZJRgABAQEASABIAAD//gBNRmlsZSBzb3VyY2U6IGh0dHA6Ly9jb21tb25zLndpa2lt
```

Comment:
you have 2 test images with ridiculously long file names?

Reply:
Yeah, they're just the same image, so their `base64` representation will be the same.